dali/operators/generic/transpose/transpose.cc

```diff
@@ -78,15 +80,18 @@ for all valid coordinates.
     .AllowSequences()
     .SupportVolumetric()
     .AddArg("perm",
-            R"code(Permutation of the dimensions of the input (e.g. [2, 0, 1]).)code", DALI_INT_VEC)
+            R"code(Permutation of the dimensions of the input, for example, [2, 0, 1].)code",
+            DALI_INT_VEC)
     .AddOptionalArg(
         "transpose_layout",
-        R"code(When set to true, the output data layout will be transposed according to perm.
-Otherwise, the input layout is copied to the output)code",
+        R"code(When set to True, the output data layout is transposed based on the perm.
+
+Otherwise, the input layout is copied to the output.)code",
```

Comment:
```suggestion
Otherwise, the input layout is copied to the output.

If ``output_layout`` is set, this argument is ignored.)code",
```
I don't think there should be a line break between "If True" and "otherwise"

Reply:
Done